### PR TITLE
Define debug method

### PR DESCRIPTION
**Pull request by awegrzyn**
@ktf `void Monitoring::debug(Metric&& metric)` prints out the metric only when `infologger://` backend is used, otherwise it does nothing. 
Let me know whether this is good enough.

### CODE REVIEW

**Comment by ktf** [comment]
The API is fine. However, I might want to send the debug information to a central store also when I use something different than infologger, e.g. if I want to debug a multi machine deployment. Why not having a "verbosity level" parameter for every backend and use that to determine whether or not sending?

**Comment by awegrzyn** [comment]
Okay, but in such case additional parameter needs to be passed to the backend url, eg: `infologger://?verbosity=debug,influxdb-udp://localhost:1234? verbosity=debug`

**Comment by ktf** [comment]
well, we could have default values, no? infologger would be "debug" by default, while influxdb would be "info" (or whatever you want to call it).

**Comment by awegrzyn** [comment]
Yes of course there will be default values, this is when you want to change default behaviour

File: src/MonitoringFactory.cxx [hunk 1]
```diff
@@ -71,6 +71,21 @@ std::unique_ptr<Backend> getFlume(http::url uri) {
   return std::make_unique<backends::Flume>(uri.host, uri.port);
 }
 
+void MonitoringFactory::SetVerbosity(std::string selected, std::unique_ptr<Backend>& backend) {
+  static const std::map<std::string, backend::Verbosity> verbisities = {
```
Comment (ktf) [inline src/MonitoringFactory.cxx]: Typo: `verbisities`.

**Comment by awegrzyn** [comment]
@ktf I think now it does what you requested

**Comment by ktf** [comment]
This looks fine for me. What’s the error about?

**Comment by awegrzyn** [comment]
I think travis just hanged at some point

**Pull request merged**